### PR TITLE
update twilio

### PR DESCRIPTION
**Pull request by nickpell**
https://github.com/twilio/twilio-python/blob/master/CHANGES.md

### CODE REVIEW

**Comment by nickpell** [comment]
@gcapalbo How do you recommend I test this?  Would like to confirm that SMS and calls work on staging before merging.

File: corehq/messaging/smsbackends/twilio/models.py [hunk 1]
```diff
@@ -47,8 +47,7 @@ def send(self, msg, orig_phone_number=None, *args, **kwargs):
                             "instances of PhoneLoadBalancingMixin")
 
         config = self.config
-        client = TwilioRestClient(config.account_sid, config.auth_token,
-            timeout=settings.SMS_GATEWAY_TIMEOUT)
+        client = Client(config.account_sid, config.auth_token)
```
Comment (gcapalbo) [inline corehq/messaging/smsbackends/twilio/models.py]: is `timeout` no longer supported?

**Comment by gcapalbo** [comment]
On Tue, Aug 22, 2017 at 4:02 PM, Nick Pellegrino <notifications@github.com>
wrote:

> @gcapalbo <https://github.com/gcapalbo> How do you recommend I test this?
> Would like to confirm that SMS and calls work on staging before merging.
>
I'll offline with you

File: corehq/messaging/smsbackends/twilio/models.py (hunk 1 above)
Comment (gcapalbo) [inline corehq/messaging/smsbackends/twilio/models.py]: (looks like it has been removed, i think that's probably fine for this one)

**Review by gcapalbo — APPROVED** [review]
looks good to me pending testing

**Comment by nickpell** [comment]
tested; looks good

**Pull request merged**